netci.groovy

```diff
@@ -14,56 +14,55 @@ def osList = ['Windows_NT', 'Windows_NT_FullFramework', 'Ubuntu14.04', 'Ubuntu16
 def configList = ['Release', 'Debug']
 
 def static getBuildJobName(def configuration, def os) {
-    return configuration.toLowerCase() + '_' + os.toLowerCase()
```

Comment:
Another example where changing the whitespace makes it super difficult to review the changes in the file.